actions/bind

```diff
@@ -173,6 +179,37 @@ def set_forwarders(forwarders):
     conf_file.close()
 
 
+def set_adblock(choice):
+    """Enable or disable ad blocking."""
+    data = [line.strip() for line in open(CONFIG_FILE, 'r')]
+    if choice == 'true':
+        # serve a single pixel gif for blocked domains
+        action_utils.webserver_enable('adblock-plinth')
+
+        # fetch ad server list
```

Comment:
Also, how about having an /etc/hosts based ad-blocking provided on FreedomBox as a service instead of using Bind or Privoxy, maybe as a feature in the configuration page?